example/lib/util/signal_level.dart

```diff
@@ -1,6 +1,26 @@
+import 'package:flutter/material.dart';
+
 enum SignalLevel { high, medium, low, unknown }
 
-SignalLevel parseRssi(int rssi) {
+extension SignalLevelExtenstion on SignalLevel {
+  Color color() {
+    switch (this) {
+      case SignalLevel.high:
+        return Colors.green;
+      case SignalLevel.medium:
+        return Colors.orange;
+      case SignalLevel.low:
+        return Colors.red;
+      case SignalLevel.unknown:
+        return Colors.grey;
+      default:
```

Comment:
Why default and unknown are different?

Reply:
Default is only introduced because of flutter analyzer warnings - all enum cases are handled in their respective `case` statements.

Reply:
Then why not do
```
case SignalLevel.unknown:
default:
    return Colors.grey;
```
?

Reply:
Seems reasonable.